build/Dockerfile

```diff
@@ -122,45 +125,62 @@ RUN apt-get install -y graphviz && \
     cd /install && \
     rm -r opencv && \
     pip3 install scikit-image && \
-    pip3 install imageio && \
-    pip3 install matplotlib
+    pip3 install imageio
 
-# installing special packages
+# install special packages
 RUN pip3 install SimpleITK && \
     pip3 install pydicom && \
     pip3 install wfdb && \
     pip3 install PyWavelets && \
     pip3 install pyedflib && \
     pip3 install segyio && \
-    pip3 install lasio
+    pip3 install lasio && \
+    pip3 install xlrd
 
 # increase counter to rebuild the image if some library changes
 ENV UPDATE_DL 1
-# installing deep learning libraries
+# install deep learning libraries
 RUN pip3 install tensorflow && \
     pip3 install --no-deps keras && \
     pip3 install --no-deps edward && \
     pip3 install torch torchvision torchtext && \
     pip3 install pyro-ppl
 
-# installing visualization libraries
+# install visualization libraries
 RUN pip3 install seaborn && \
```

Comment:
Let's combine image and vis libraries as they are extremely linked (e.g. scikit-image requires matplotlib).
Please check the order of packages as they required one another.